Collections/Student.java

```diff
@@ -0,0 +1,50 @@
+public class Student {
+    private int id;
+    private String name;
+    private int age;
+    private int grade;
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
+        this.id = id;
+    }
+
+    public Student() {

```

Comment:
No need to have empty constructor if you don't use it in the code.

Reply:
I use it in ArrayListMap.java:
...
Student student = new Student();
student = studentMap.get(4);
...